packages/design-system/package.json

```diff
@@ -46,61 +40,67 @@
     "preview": "pnpm ladle serve",
     "test": "pnpm vitest --dir=src"
   },
+  "peerDependencies": {
+    "@react-aria/collections": "3.0.0-alpha.5",
+    "@react-aria/dnd": "3.7.4",
+    "@react-aria/focus": "3.18.4",
+    "@react-aria/interactions": "3.22.4",
+    "@react-aria/overlays": "3.23.4",
+    "@react-aria/ssr": "3.9.6",
+    "@react-aria/utils": "3.25.3",
+    "@react-stately/data": "3.11.7",
+    "@react-stately/overlays": "3.6.11",
+    "@react-stately/utils": "3.10.4",
+    "@react-types/shared": "3.25.0",
+    "react": "18.3.1",
```

Comment:
was the v19 rc version removed in order to resolve the TS error you were seeing for now?